### ISSUE TITLE
Empty title and detail in Toolbar.

### ISSUE THREAD

**Issue by TheHmmka** [CosmicMind/Material#381]
When you don't set leftControl or rightControl in Toolbar you can't see any title.

```
self.navigationBarView = [[Toolbar alloc] initWithLeftControls: nil rightControls: nil];
self.navigationBarView.titleLabel.font = SAN_FRANCISCO_REGULAR_17;
self.navigationBarView.titleLabel.textColor = [UIColor whiteColor];
	
self.navigationBarView.detailLabel.font = SAN_FRANCISCO_REGULAR_12;
self.navigationBarView.detailLabel.textColor = [UIColor whiteColor];
	
self.navigationBarView.backgroundColor = [[DataManager sharedInstance] getColorForCurrentSection];
self.navigationBarView.contentInset = UIEdgeInsetsMake(22.0, 6.0, 6.0, 6.0);
self.navigationBarView.spacing = 10.0;
[self.view addSubview: self.navigationBarView];
```

Then I try to set title with code, but Toolbar is empty:
`self.navigationBarView.titleLabel.text = NSLocalizedString(@"Settings section title", nil);`

And when I set leftControls (for test) title appears:
```
FlatButton* playlistsButton = [FlatButton new];
playlistsButton.pulseColor = [UIColor whiteColor];
[playlistsButton setImage: [UIImage imageNamed: @"TagsIcon"] forState: UIControlStateNormal];
[playlistsButton addTarget: self action: @selector(playlistsButtonTouchSelector) forControlEvents: UIControlEventTouchUpInside];
self.navigationBarView.leftControls = @[playlistsButton];
```

<img width="410" alt="2016-05-16 09 52 16" src="https://cloud.githubusercontent.com/assets/468660/15282454/1bfac750-1b4c-11e6-9884-6d37a50617a1.png">
<img width="410" alt="2016-05-16 09 52 35" src="https://cloud.githubusercontent.com/assets/468660/15282459/1f5d4300-1b4c-11e6-9cc8-c5a69c28c9f4.png">

**Comment by mohpor** [comment]
Let me check.

**Comment by mohpor** [comment]
I have checked on this.  I could reproduce it.
A quick solution is to call `prepareView()` on your toolbar at some point (before or after adding to your view).

**Comment by TheHmmka** [comment]
Thanks, will try.

**Comment by mohpor** [comment]
I have tested it, it should work. I will work on fix though.

**Comment by danieldahan** [comment]
@TheHmmka I have fixed the issue. Thank you @mohpor for the investigation :) Later today a release will be issued.

**Comment by TheHmmka** [comment]
Very nice! Thank you!

**Comment by danieldahan** [comment]
Should be fixed in [1.41.2](https://github.com/CosmicMind/Material/releases/tag/1.41.2).
